test/utils/utils.go

```diff
@@ -112,7 +112,7 @@ func GetWithTimeout(
 		}
 
 		return nil
-	}, timeout, 1).ShouldNot(HaveOccurred())
+	}, timeout, 1).Should(BeNil())
```

Comment:
Why was this changed? I thinking using `ShouldNot(HaveOccurred())` with errors might be a best practice.